cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -59,6 +61,10 @@
 
     public static final String ENV_VAR_TLS_ENABLED = "STRIMZI_TLS_ENABLED";
 
+    public static final String ENV_VAR_CLUSTER_CA_CERT_SECRET_DATA = "STRIMZI_CLUSTER_CA_CERT_SECRET_DATA";
+    public static final String ENV_VAR_EO_KEY_SECRET_DATA = "STRIMZI_EO_KEY_DATA";
+    public static final String ENV_VAR_EO_CERTIFICATE_CHAIN_DATA = "STRIMZI_EO_CERTIFICATE_CHAIN_DATA";
+
```

Comment:
The `DATA` or `SECRET_DATA` sounds a bit weird in the name. Why don't you follow the same naming you used in the TO code? `STRIMZI_PRIVATE_KEY`, `STRIMZI_CERTIFICATE_CHAIN`, `STRIMZI_TRUSTED_CERTIFICATES`.

Reply:
+1. Conceptually there is a very important distinction between a certificate that ended up in a trust store and the certificates in the chain. They're all certificates, but the former implies a trust relationship, whereas the latter do not. Ensuring the code that manipulates certs maintains this distinction is important. So using `trust` in variable and constant names is a good idea.